cmd/kogito/command/install/kogito_operator_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/controller/kogitoapp/resource"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/operator"
+	"testing"
+
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/test"
+
+	"github.com/stretchr/testify/assert"
+
+	apps "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_InstallOperator(t *testing.T) {
+	cli := fmt.Sprintf("install operator -p kogito")
+	test.SetupCliTest(cli, context.CommandFactory{BuildCommands: BuildCommands}, &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "kogito"}})
+
+	lines, _, err := test.ExecuteCli()
+	assert.NoError(t, err)
+	assert.Contains(t, lines, "Kogito Operator successfully deployed")
+}
+
+func Test_InstallOperatorNoNamespace(t *testing.T) {
+	cli := fmt.Sprintf("install operator -p kogito")
+	test.SetupCliTest(cli, context.CommandFactory{BuildCommands: BuildCommands}, &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "kogito"}})
```

Comment:
Why is this command tested? It seems to be a copy of the test above.

Reply:
This is to have the `kogito` project saved in the config file mentioned above. In this test, we call the install command twice just to verify this flag.

Reply:
ok